tests/acceptance/features/apiSharingNg1/removeAccessToDrive.feature

```diff
@@ -246,3 +246,19 @@ Feature: Remove access to a drive
     When user "Alice" removes the last link share of space "projectSpace" using permissions endpoint of the Graph API
     Then the HTTP status code should be "204"
     And user "Alice" should not have any "link" permissions on space "projectSpace"
+
+  @env-config
+  Scenario: delete space share after the share role Space Editor Without Versions has been disabled
+    Given using spaces DAV path
+    And the administrator has enabled the permissions role "Space Editor Without Versions"
+    And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "new-space" with the default quota using the Graph API
+    And user "Alice" has sent the following space share invitation:
+      | space           | new-space                     |
+      | sharee          | Brian                         |
+      | shareType       | user                          |
+      | permissionsRole | Space Editor Without Versions |
+    And the administrator has disabled the permissions role "Space Editor Without Versions"
```

Comment:
What happens if this step is skipped? Are there alternative outcomes? If so, have we covered them? If not, why is this step crucial?

Reply:
The outcome would be same if the disabling step is skipped. These scenarios are to check the outcomes of the actions done after the permissions role of the shared resources is disabled.